src/tests/Microsoft.PowerFx.Interpreter.Tests/SandboxTests.cs

```diff
@@ -30,7 +30,7 @@ public async void MaxRecursionDepthTest()
         {
             var config = new PowerFxConfig(null)
             {
-                MaxCallDepth = 5
+                MaxCallDepth = 15
```

Comment:
>MaxCallDepth = 15 [](http://example.com/codeflow?start=16&length=17)

This is because of the "Expansion" and injecting additional calls?

Reply:
correct